src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -43,22 +55,54 @@ describe('generateConfluenceMarkdown function', () => {
         expect(results).toContain(fakeIssues[0].title);
         expect(results).toContain(fakeIssues[0].title);
     });
-    //      it('should return two dependencies, each with their issue', () => {
-    //         const depsMap: Map<string, Issue[]> = new Map();
-    //         const a = generateConfluenceMarkdown(dependency);
-    //      });
-    //      it('should pass in an issue that is too old', () => {
-    //
-    //          // expect().toContain();
-    //      });
-    //       it('should return an issue that has square brackets and curly braces in its title', () => {
-    //        // failing test
-
-    //           expect(generateConfluenceMarkdown(a, b))
-    //     });
-    //       it('should pass in a dependency with one issue and make sure the all fields for that issue are correct', () => {
-    //        // failing test
-
-    //           expect(generateConfluenceMarkdown(a, b))
-    //     });
+    it('should return two dependencies, each with their issue', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency1 = 'Graphql';
+        const dependency2 = 'TypeStrong';
+
+        mockDependencyMap.set(dependency1, fakeIssues);
+        mockDependencyMap.set(dependency2, fakeIssues2);
+
+        const results = generateConfluenceMarkdown(mockDependencyMap);
+        expect(results).toContain(dependency1);
+        expect(results).toContain(fakeIssues[0].title);
+        expect(results).toContain(fakeIssues[1].title);
+        expect(results).toContain(dependency2);
+        expect(results).toContain(fakeIssues2[0].title);
+    });
+    it('should return an issue that has square brackets and curly braces in its title', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency = 'React';
+
+        fakeIssues2[0].title = '[Navigation Editor] Dropdown menus too narrow {}';
+        mockDependencyMap.set(dependency, fakeIssues2);
+
+        const results = generateConfluenceMarkdown(mockDependencyMap);
+        expect(results).toContain(fakeIssues2[0].title);
+    });
+    it('should pass in a dependency with one issue and make sure the all fields for that issue are correct', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency = 'OSS';
+
+        fakeIssues2[0].title = 'Added more info to readme.md';
+        mockDependencyMap.set(dependency, fakeIssues2);
+
+        const results = generateConfluenceMarkdown(mockDependencyMap);
+
+        expect(results).toContain(fakeIssues2[0].title);
```

Comment:
It would be good for the expect to include the | on either side of these fields, to make sure no additional data got inserted.